### PR TITLE
chore: add new publish script

### PR DESCRIPTION
**Pull request by MethodenMann**
- lerna automatically did release all changed packages. This was not desired
  since we want to publish specific packages at a time. Hence a custom
  publish script is added with changelog capabilities.
- add .yarnrc for all public packages and for plop

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,14 +98,21 @@ npm run plop
 At the moment there is a generator to create a react-component and a generator to add a redux-action.
 
 ## Publish bundle
-Once the package is ready to publish, run following npm scripts. This registers the bundle
-in the npm registry, increases versions and creates a git tag using lerna publish command.
+Once the package is ready to publish, run the publish-package npm scripts.
+This will  run yarn setup at the beginning to make sure all local dependencies are up to date.
```
Comment (rzueger) [inline README.md]: human linter: 2 spaces between "will" and "run"

File: README.md [hunk 2]
```diff
@@ -98,14 +98,21 @@ npm run plop
 At the moment there is a generator to create a react-component and a generator to add a redux-action.
 
 ## Publish bundle
-Once the package is ready to publish, run following npm scripts. This registers the bundle
-in the npm registry, increases versions and creates a git tag using lerna publish command.
+Once the package is ready to publish, run the publish-package npm scripts.
+This will  run yarn setup at the beginning to make sure all local dependencies are up to date. 
+After that the new version can be entered. The script will then create a changelog entry which can be edited manually,
+pumps the version of the package, creates a git release tag and uploads the package to the npm registry.
```
Comment (rzueger) [inline README.md]: "pumps" -> "bumps"

File: README.md [hunk 3]
```diff
@@ -98,14 +98,21 @@ npm run plop
...
 ```
-yarn publish-packages
+yarn publish-package {PACKAGE_NAME}
 ```
 
-For more information see [the official documentation](https://github.com/lerna/lerna/tree/master/commands/publish#readme) 
+To check the upcoming changelog:
+
+```
+yarn changelog {PACKAGE_NAME}
```
Comment (rzueger) [inline README.md]: nice:)

File: README.md [hunk 4]
```diff
@@ -156,14 +164,21 @@ First line must not be longer than 70 characters, second line is always blank an
...
 - If changes affect the monorepo itself, the scope is set to 'tocco-client'.
 
 #### Message body
+- If a release relevant feature is added with the commit, a **Changelog:** line can be added.
+  e.g.
+  ```
+  Changelog: Add feature XY
+  ```
+  A commit  can have several changelog entries but an entry needs to be on one line.
+  The changelog entries are used to generate the changelog of the corresponding `<scope>` package
```
Comment (rzueger) [inline README.md]: I'd also add a hint that it has to be a single line

Comment (rzueger) [reply README.md]: another error found by the human linter: 2 spaces between "commit" and "can"

File: scripts/publish-package.sh [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    touch ${changelog_file}
+fi
+
+(echo  -e "${new_version}\n${changelog}\n" && cat "${changelog_file}") > "${tmp_file}" && mv "${tmp_file}" "${changelog_file}"
+read -p "${color_green}Edit the changelog and press ENTER to continue${color_reset}"
+
+git commit -m "docs(${package}): changelog ${new_version}" ${changelog_file}
+echo "publishing ${package} with version ${new_version}"
+yarn publish --new-version ${new_version}
+echo "${color_green} Published! Now you push!${color_reset}"
```
Comment (khanlifts) [inline scripts/publish-package.sh]: Spacing in line 31 before Published.

**Pull request merged**